google-cloud-logging/lib/google/cloud/logging/logger.rb

```diff
@@ -54,10 +66,9 @@ class Logger
         attr_reader :labels
 
         ##
-        # A OrderedHash of Thread IDs to Stackdriver request trace ID. The
-        # Stackdriver trace ID is a shared request identifier across all
-        # Stackdriver services.
-        attr_reader :trace_ids
+        # A OrderedHash of Thread IDs to RequestInfo, keeping track of data
+        # associated with the request being handled by the current thread.
+        attr_reader :request_info
```

Comment:
Ah good point. In the interest of minimizing the blast radius for this change, I'll roll the API back.